### PR TITLE
add `plotInset` parameter to style

### LINKED ISSUE #406: Add `plotInset` parameter to style
https://nbviewer.org/github/JetBrains/lets-plot-kotlin/blob/master/docs/examples/jupyter-notebooks/f-4.7.1/theme_plot_inset.ipynb

### PR DESCRIPTION
**Pull request by lucas-watkins**
Try to close #406

### CODE REVIEW

**Comment by AndreiKingsley** [comment]
Looks like `Inset` class is a basically the same as `Margin`. So, may be we don't need another class here, only property/function names.

File: kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/style/CustomStyle.kt [hunk 1]
```diff
@@ -70,6 +70,33 @@ public interface WithMargin {
     }
 }
 
+public data class Inset(val top: Double, val right: Double, val bottom: Double, val left: Double) {
```
Comment (AndreiKingsley) [inline kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/style/CustomStyle.kt:73]: Just the same as `Margin`

**Review by AndreiKingsley — CHANGES_REQUESTED** [review]

File: kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/style/CustomStyle.kt (hunk 1 above)
Comment (lucas-watkins) [reply kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/style/CustomStyle.kt:73]: Will do, would a typealias be acceptable here?

Comment (AndreiKingsley) [reply kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/style/CustomStyle.kt]: Yes

**Review by AndreiKingsley — APPROVED** [review]

**Pull request merged**